components/TxConfirmationDialog.qml

```diff
@@ -0,0 +1,458 @@
+            StateChangeScript { script: backButton.forceActiveFocus() }
+        }, State {
+            // creating or sending transaction, show tx details and don't show any button
+            name: "bottomText";
+            when: errorText.text =="" && bottomText.text !==""
+            PropertyChanges { target: errorText; visible: false }
+            PropertyChanges { target: txAmountText; visible: root.transactionAmount !== "(all)" || (root.transactionAmount === "(all)" && currentWallet.isHwBacked() === true) }
+            PropertyChanges { target: txAmountBusyIndicator; visible: !txAmountText.visible }
+            PropertyChanges { target: txFiatAmountText; visible: txAmountText.visible && persistentSettings.fiatPriceEnabled && root.transactionAmount !== "(all)" }
+            PropertyChanges { target: txDetails; visible: true }
+            PropertyChanges { target: bottom; visible: true }
+            PropertyChanges { target: bottomMessage; visible: true }
+            PropertyChanges { target: buttons; visible: false }
+        }
+    ]
+
+    // same signals as Dialog has
+    signal accepted()
+    signal rejected()
+    signal closeCallback();
+
+    function open(description, amount, address, priority) {
+        if (description) {
+            root.transactionDescription = description;  
+        }
+        if (amount) {
+            root.transactionAmount = amount;  
+        }
+        if (address) {
+            root.transactionAddress = address;  
+        }
+        if (priority) {
+            root.transactionPriority = priority;  
+        }
+
+        root.visible = true;
+        
+        //clean previous error message
+        errorText.text = "";        
+    }
+    
+    function close() {
+        root.visible = false;
+        closeCallback();
+    }
+
+    function clearFields() {
+        root.transactionAmount = "";
+        root.transactionAddress = "";
+        root.transactionDescription = "";
+        root.transactionFee = "";
+        root.transactionPriority = "";
+        root.transactionID = "";
+    }
+
+    function showFiatConversion(valueXMR) {
+        return (fiatApiConvertToFiat(valueXMR) === "0.00" ? "<0.01 " + fiatApiCurrencySymbol() : "~" + fiatApiConvertToFiat(valueXMR) + " " + fiatApiCurrencySymbol());
+    }
+
+    ColumnLayout {
+        spacing: 10
+        anchors.fill: parent
+        anchors.margins: 25
+
+        RowLayout {
+            Layout.topMargin: 10
+            Layout.fillWidth: true
+
+            MoneroComponents.Label {
+                id: dialogTitle
+                Layout.fillWidth: true
+                fontSize: 18
+                fontFamily: "Arial"
+                horizontalAlignment: Text.AlignHCenter
+                text: {
+                    if (appWindow.viewOnly) {
+                        "Create transaction file" + translationManager.emptyString
+                    } else if (appWindow.sweepUnmixable) {
```

Comment:
This does not look correct, there is no global sweepUnmixable property var. Also please use a local property var here instead.